
R version 4.3.2 (2023-10-31) -- "Eye Holes"
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: x86_64-pc-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type 'license()' or 'licence()' for distribution details.

  Natural language support but running in an English locale

R is a collaborative project with many contributors.
Type 'contributors()' for more information and
'citation()' on how to cite R or R packages in publications.

Type 'demo()' for some demos, 'help()' for on-line help, or
'help.start()' for an HTML browser interface to help.
Type 'q()' to quit R.

> 
> 
> .rslurm_func <- readRDS('f.RDS')
> .rslurm_params <- readRDS('params.RDS')
> .rslurm_more_args <- readRDS('more_args.RDS')
> .rslurm_id <- as.numeric(Sys.getenv('SLURM_ARRAY_TASK_ID'))
> .rslurm_istart <- .rslurm_id * 1985 + 1
> .rslurm_iend <- min((.rslurm_id + 1) * 1985, nrow(.rslurm_params))
> .rslurm_result <- do.call(parallel::mcmapply, c(
+     FUN = .rslurm_func,
+     .rslurm_params[.rslurm_istart:.rslurm_iend, , drop = FALSE],
+     MoreArgs = list(.rslurm_more_args),
+     mc.cores = 52,
+     mc.preschedule = TRUE,
+     SIMPLIFY = FALSE))
> 
> saveRDS(.rslurm_result, file = paste0('results_', .rslurm_id, '.RDS'))
> 
> proc.time()
    user   system  elapsed 
  19.589    5.466 1714.666 
fnd-MUR-GLOB-v02.0-fv04.1.nc?sd_name=analysed_sst&a_srs=EPSG:4326&a_scale=0.001&a_offset=25' does not exist in the file system, and is not recognized as a supported dataset name.
input source not readable: vrt:///vsicurl/https://archive.podaac.earthdata.nasa.gov/podaac-ops-cumulus-protected/MUR-JPL-L4-GLOB-v4.1/20221109090000-JPL-L4_GHRSST-SSTfnd-MUR-GLOB-v02.0-fv04.1.nc?sd_name=analysed_sst&a_srs=EPSG:4326&a_scale=0.001&a_offset=25
ERROR 4: `vrt:///vsicurl/https://archive.podaac.earthdata.nasa.gov/podaac-ops-cumulus-protected/MUR-JPL-L4-GLOB-v4.1/20210220090000-JPL-L4_GHRSST-SSTfnd-MUR-GLOB-v02.0-fv04.1.nc?sd_name=analysed_sst&a_srs=EPSG:4326&a_scale=0.001&a_offset=25' does not exist in the file system, and is not recognized as a supported dataset name.
input source not readable: vrt:///vsicurl/https://archive.podaac.earthdata.nasa.gov/podaac-ops-cumulus-protected/MUR-JPL-L4-GLOB-v4.1/20210220090000-JPL-L4_GHRSST-SSTfnd-MUR-GLOB-v02.0-fv04.1.nc?sd_name=analysed_sst&a_srs=EPSG:4326&a_scale=0.001&a_offset=25
ERROR 4: `vrt:///vsicurl/https://archive.podaac.earthdata.nasa.gov/podaac-ops-cumulus-protected/MUR-JPL-L4-GLOB-v4.1/20210221090000-JPL-L4_GHRSST-SSTfnd-MUR-GLOB-v02.0-fv04.1.nc?sd_name=analysed_sst&a_srs=EPSG:4326&a_scale=0.001&a_offset=25' does not exist in the file system, and is not recognized as a supported dataset name.
input source not readable: vrt:///vsicurl/https://archive.podaac.earthdata.nasa.gov/podaac-ops-cumulus-protected/MUR-JPL-L4-GLOB-v4.1/20210221090000-JPL-L4_GHRSST-SSTfnd-MUR-GLOB-v02.0-fv04.1.nc?sd_name=analysed_sst&a_srs=EPSG:4326&a_scale=0.001&a_offset=25
Warning message:
In mclapply(seq_len(n), do_one, mc.preschedule = mc.preschedule,  :
  scheduled cores 5, 2, 3 encountered errors in user code, all values of the jobs will be affected
> 
> saveRDS(.rslurm_result, file = paste0('results_', .rslurm_id, '.RDS'))
> 
> proc.time()
    user   system  elapsed 
 881.551   48.845 3883.643 
